src/app/about/page.tsx

```diff
@@ -1,92 +1,76 @@
-          genomic and related phenotypic and clinical data to improve research,
-          policymaking and healthcare across Europe. The GDI project aims to
-          unlock a data network of over one million genome sequences for
-          research and clinical reference. This will create unprecedented
-          opportunities for transnational and multi-stakeholder actions in
-          personalised medicine for cancer, common, rare and infectious diseases
-          as well as access to a reference genome collection representing the
-          European population (Genome of Europe).
-        </p>
-      </p>
+  const [response, setResponse] = useState<AboutResponse>({
+    status: "loading",
+  });
 
-      <p className="mb-8">
-        <PageSubHeading className="my-4">User Portal</PageSubHeading>
-        <p>
-          The User Portal, developed by the Genomic Data Infrastructure (GDI)
-          project, is the central entry point for accessing genomic data. As
-          part of the Genomic Data Infrastructure (GDI) project, it unlocks a
-          vast repository of over one million genome sequences, this platform is
-          currently under development and will serve as the main European-level
-          hub for data access, providing a user-friendly interface for
-          researchers and healthcare professionals.{" "}
-        </p>
-      </p>
+  useEffect(() => {
+    const fetchMarkdown = async () => {
+      try {
+        setResponse({ status: "loading" });
+        const response = await fetch("/about.md");
+        const text = await response.text();
 
-      <p className="mb-8">
-        <PageSubHeading className="my-4">Key Objectives</PageSubHeading>
-        <ul className="list-inside list-disc">
-          <li>
-            To link and to provide cross-border access to genomic and related
-            phenotypic datasets across Europe
-          </li>
-          <li>
-            To advance understanding of genomics for more precise and faster
-            clinical decision making, diagnostics, treatments systems, and to
-            benefit the overall economy
-          </li>
-          <li>
-            To align with the development under the European Health Data Space
-            (EHDS)
-          </li>
-          <li>
-            To Facilitate research, policy-making, and healthcare improvements
-          </li>
-          <li>
-            To maintain awareness, acceptance and trust in the main groups of
-            stakeholders, notably European citizens, data holders, healthcare
-            professionals, researchers and public health authorities
-          </li>
-        </ul>
-      </p>
+        // @ts-expect-error" Type assertion is necessary because our custom renderer is a partial
+        // implementation of the Renderer interface, but we know it's compatible for our use case.
+        marked.use({ renderer: renderer as Partial<marked.Renderer> });
```

Comment:
question: why do we need to have a special renderer?

Reply:
to show the tags content based on what we want to show... with styles and ...